examples/monitoring-alert/variables.tf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
consider allowing user to specify a region.  we can use a default (i.e what is already hardcoded in the main is a fine region)

Reply:
In the next PR I can add regions, since the kms module will accept a region as input.
The secret manager module itself doesn't accept a region as input, so it may not make sense to add regions to this PR.